### PR TITLE
Navigation Created

### PR DESCRIPTION
**Pull request by this8**
Review changes and merge.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -8,7 +8,7 @@
     <!-- Bootstrap CSS -->
     <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css" integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO" crossorigin="anonymous">
 
-    <title>Hello, world!</title>
+    <title>Web Assignment</title>
```
Comment (this8) [inline index.html:11]: blah blah blah

File: index.html [hunk 2]
```diff
@@ -54,6 +54,6 @@
     <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/js/bootstrap.min.js" integrity="sha384-ChfqqxuZUCnJSK3+MXmPNIyE6ZbWh2IMqE241rYiqJxyMiZ6OW/JmZQ5stwEULTy" crossorigin="anonymous"></script>
 
     <!--footer-->
-    <div class="fixed-bottom">Hi!</div>
```
Comment (this8) [inline index.html:57]: add this

File: index.html (hunk 1 above)
Comment (wphkweerasinghe) [inline index.html:11]: change the title

Comment (this8) [reply index.html:11]: ok

File: index.html (hunk 2 above)
Comment (this8) [reply index.html:57]: done

**Pull request merged**